packages/lib/src/api-command.ts

```diff
@@ -28,6 +28,9 @@ export abstract class APICommand<T extends typeof APICommand.flags> extends Smar
 		}),
 	}
 
+	static apiDocsURL = (name: string): string => '\nFor API information, see:\n\n' +
```

Comment:
I love that this way there's only one place to update when/if the URL changes in the future. Or if oclif changes how it formats stuff.